### ISSUE TITLE
how resize chat window?

### ISSUE THREAD

**Issue by redboyke** [FailcoderAddons/supervillain-ui#98]
i dont know how i can resize the chat window in SV-chat 
it shows a small square and i want to adjust the chat window so i can read it.
![2017-06-24 17_38_55-world of warcraft](https://user-images.githubusercontent.com/1980758/27509754-11dc14ee-5904-11e7-94b1-b6d16adf6875.png)

**Comment by finalsliver** [comment]
Under docks in the SVui settings. change the size variables to alter the chat box dimensions
![docks](http://i.imgur.com/gfI2NsR.png)

Under Fonts, then chat to adjust font type and size to your liking.
![Fonts](http://i.imgur.com/ZuBZQZy.png)

**Comment by redboyke** [comment]
ok thank you

**Comment by finalsliver** [comment]
come back to us if I missed something. if not we can close this issue :)

**Comment by redboyke** [comment]
i can resize it but i have a problem with the chat that if im talking in party chat that when i press enter it goes back to say and the profile doesnt remember my different tabs like W for whispers and P for party and  my T for trade when i make a new char i have to make them again.

**Comment by finalsliver** [comment]
also seeing some odd interaction with chat windows on log out. Can't seem to get them set up on multi characters either without manually hopping over. these tend to need to be reset on log in to work, sometimes not working at all...

@joeymagz can you take a look at this one?

Character 1
Set up new chat window, set to only show looted items. 
Set up new chat window, Set to only show Whispers/officerchat/blizzwhispers

Log out, then back in. check if tabs are working (also notice the name disapears?)
Restart game (still not working and names disapear.

TL:DR; 
extra chat windows don't appear to save properly.
Possible enhancement. allow chat windows to be copied to other characters.
